backend/backend/views.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import viewsets
+from rest_framework.response import Response
+import requests
+import json
+import os
+
+
+class OAuthCallbackViewSet(viewsets.ViewSet):
+    authentication_classes = []
+    permission_classes = []
+
+    def list(self, request):
+        CLIENT_ID = os.getenv("CLIENT_ID")
+        CLIENT_SECRET = os.getenv("CLIENT_SECRET")
```

Comment:
Quick Question: Would sending two different users the same `client_id` and `client_secret` be the appropriate thing to do?

Reply:
@ceguiluzrosas From what I've read, yes. Client ID/Client Secret is application specific. Since we're only running one application (our API), having a single id/secret is appropriate. In the case of our code, we won't have to provide the Client Secret to users at all, only the Client ID. The way this is now set up, Client Secret is handled entirely in the backend, which seems to be typical.